### ISSUE TITLE
Concider seeting a default timeout

### ISSUE THREAD

**Issue by fnordfish** [customerio/customerio-ruby#18]
`Customerio::Client` does not set a `default_timeout` causing `HTTParty` to try to connect to the API endpoint "forever" if there are networking problems.

My current workaround is something like that:

```ruby
klass = Customerio::Client.tap {|klass|
  klass.default_options.reverse_merge!(
    timeout: 1.0
  )
}

klass.new(...).identify(...)
```

**Comment by stayhero** [comment]
:+1:

**Comment by yjukaku** [comment]
:+1:

**Comment by alisdair** [comment]
Fixed by #24. Gem release coming soon.
